Dockerfile

```diff
@@ -73,7 +74,11 @@ RUN addgroup --system --gid ${FOUNDRY_UID} foundry \
   && npm install && echo ${VERSION} > image_version.txt
 
 VOLUME ["/data"]
+# HTTP Server
 EXPOSE 30000/TCP
+# TURN Server
+EXPOSE 33478/UDP
+EXPOSE 49152-65535/UDP
```

Comment:
Docker containers have performance issues when opening ranges of ports this big. See coturn docker image issue [#3](https://github.com/instrumentisto/coturn-docker-image/issues/3). This should be specified by the user via their docker-compose.yml file if they want this or use `network-mode: host` or `--network-mode=host`

Reply:
Thanks for pointing this out.  I was unaware.   

I think we should ok here, because this is just exposing the port, we're not publishing it.

Reply:
I was wrong.   This is causing issues for at least one user of the Portainer front end.  It seems that it is far too easy to publish all EXPOSEd ports using that tool.  
![unknown](https://user-images.githubusercontent.com/3229435/97597093-d52d9a80-19db-11eb-98e8-40c4b63c5a7e.png)

Since unexposed ports can still be published I'm going to roll back this change.